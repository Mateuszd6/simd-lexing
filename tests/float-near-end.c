int main() int int int int int int int float f = 1231298.91328982391;
