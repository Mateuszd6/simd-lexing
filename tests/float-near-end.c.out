./tests/float-near-end.c:1:1: TOK 0 (L = 3): "int"
./tests/float-near-end.c:1:5: TOK 0 (L = 4): "main"
./tests/float-near-end.c:1:9: TOK 1 (L = 1): "("
./tests/float-near-end.c:1:10: TOK 1 (L = 1): ")"
./tests/float-near-end.c:1:12: TOK 0 (L = 3): "int"
./tests/float-near-end.c:1:16: TOK 0 (L = 3): "int"
./tests/float-near-end.c:1:20: TOK 0 (L = 3): "int"
./tests/float-near-end.c:1:24: TOK 0 (L = 3): "int"
./tests/float-near-end.c:1:28: TOK 0 (L = 3): "int"
./tests/float-near-end.c:1:32: TOK 0 (L = 3): "int"
./tests/float-near-end.c:1:36: TOK 0 (L = 3): "int"
./tests/float-near-end.c:1:40: TOK 0 (L = 5): "float"
./tests/float-near-end.c:1:46: TOK 0 (L = 1): "f"
./tests/float-near-end.c:1:48: TOK 1 (L = 1): "="
./tests/float-near-end.c:1:57: TOK 3 (L = 19): "1231298.91328982391"
./tests/float-near-end.c:1:76: TOK 1 (L = 1): ";"
Parsed: 2 lines, 11 ids, 0 strings, 0 chars, 0 ints, 0 hex,   1 floats,    0 //s, 0 /**/s,   0 #foo
